### ISSUE TITLE
Add color

### ISSUE THREAD

**Issue by GaliShapira** [ji-devs/ji-cloud#2223]
1. Place the popup bigger, it can start higher than the color button. I will give tomorrow maximum high for the popup.
<img width="587" alt="Screen Shot 2022-01-10 at 12 59 10" src="https://user-images.githubusercontent.com/88834002/148755388-443c58d7-596b-4d76-a34a-dde72a25a4f0.png">

2. When using the picker to add a new color, there is no button to add the color. User cannot add the color
<img width="254" alt="Screen Shot 2022-01-10 at 12 58 49" src="https://user-images.githubusercontent.com/88834002/148755421-2b9274ee-b35d-4a9d-af04-b30e2bd6948b.png">
.

**Comment by MendyBerger** [comment]
@corinnewo MVP?

**Comment by corinnewo** [comment]
@MendyBerger Yes, for MVP - you had said that you were implementing David's overlays.

**Comment by MendyBerger** [comment]
Resolved
